CHANGELOG.md

```diff
@@ -8,8 +8,19 @@ This file was introduced starting Kani 0.23.0, so it only contains changes from
 
 ### Breaking Changes
 
+* Set minimum python to 3.7 in docker container and release action by @remi-delmas-3000 in https://github.com/model-checking/kani/pull/2879
 * Delete `any_slice` which has been deprecated since Kani 0.38.0. by @zhassan-aws in https://github.com/model-checking/kani/pull/2860
 
+### What's Changed
+
+* Make `cover` const by @jswrenn in https://github.com/model-checking/kani/pull/2867
+* Change `expect()` from taking formatted strings to use `unwrap_or_else()` by @matthiaskrgr in https://github.com/model-checking/kani/pull/2865
+* Fix setup for `aarch64-unknown-linux-gnu` platform by @adpaco-aws in https://github.com/model-checking/kani/pull/2864
+* Do not override `std` library during playback by @celinval in https://github.com/model-checking/kani/pull/2852
+* Update Rust toolchain to 2023-11-06 by @zhassan-aws in https://github.com/model-checking/kani/pull/2858
```

Comment:
The toolchain date is 2023-11-11. You can use this format:

Rust toolchain upgraded to <date> by <contributor>+ where <date> is the latest date and <contributor>+ are the human contributors (i.e., exclude automatic upgrades).